### ISSUE TITLE
Can't resolve '../constants/patterns'

### ISSUE THREAD

**Issue by briangonzalezmia** [djorg83/react-bootstrap-sweetalert#54]
I just upgraded to 5.1.3 and am getting this error below when I try to run Webpack dev server. I have already deleted all node modules and re-ran npm install and no go.  Any suggestions? Thanks, B!!!

```
 ERROR #98123  WEBPACK

Generating development JavaScript bundle failed

Can't resolve '../constants/patterns' in '{...}/node_modules/react-bootstrap-sweetalert/dist/components'

File: node_modules/react-bootstrap-sweetalert/dist/components/SweetAlert.js

failed Building development bundle - 11.993s

```

**Comment by brunnomatheuus** [comment]
I am also experiencing this problem

**Comment by djorg83** [comment]
Must be missing in the published tar.  I'll fix this tonight when I get home.

**Comment by djorg83** [comment]
Isssue resolved with v5.1.4
